src/context/BehandlingerContext.js

```diff
@@ -29,9 +31,70 @@ export const BehandlingerProvider = ({ children }) => {
         const valgtBehandling = behandlinger.find(
             b => b.behandlingsId === behandling.behandlingsId
         );
+
         setValgtBehandling(valgtBehandling);
     };
 
+    const fetchBehandlingerMedPersoninfo = async () => {
+        const alleBehandlinger = await fetchAlleBehandlinger();
+        const behandlingerMedPersoninfo = await hentNavnForBehandlinger(alleBehandlinger);
```

Comment:
Her kan `alleBehandlinger` være undefined, hvis fetchen feilet, feks ved 404 eller 401, fordi catch-blokken i `fetchAlleBehandlinger` aldri returnerer noe, da resolves `await fetchAlleBehandlinger` til undefined. Med andre ord blir ikke promiset rejected. Jeg holder på å undersøke hva som er bra feilhåndteringsoppsett for slike lenkede promise-kall, det er litt kilent.

Reply:
Kan den ikke returnere `[]`? Erroren blir jo satt

Reply:
Catch-blokken returnerer ingenting, da vil await resolve med `undefined` som verdi.

Du kan teste det ved å kjøre `fetchAlleBehandlinger().then(() => console.log('ok'), () => console.log('feilet'));` i `fetchBehandlingerMedPersoninfo` og la serveren returnere 404. Da logges det "ok" i konsollet.

Reply:
Rett skal være rett, det vil jo ikke skje noe funksjonelt feil i slike situasjoner, bare logges en feilmelding fordi koden kjører videre :) Men jeg tenker det er greit å ha kontrollen på feilsituasjoner.